test/land/create_bkg_ens.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e
+################################################
+# 4. CREATE BACKGROUND ENSEMBLE (LETKFOI)
+################################################
+YYYY=2021
+MM=03
+DD=23
+HH=18
+FILEDATE=$YYYY$MM$DD.${HH}0000
+
+project_binary_dir=$1
+project_source_dir=$2
+
+echo "project_binary_dir: $project_binary_dir"
+echo "project_source_dir: $project_source_dir"
+
+##module use $project_source_dir/modulefiles
+##module load GDAS/hera
+echo "GDASAPP_TESTDATA: $GDASAPP_TESTDATA"
+
+RSTDIR=$GDASAPP_TESTDATA/lowres/gdas.$YYYY$MM$DD/12/atmos/RESTART/
```

Comment:
use `date` to get 12 from $HH?

Reply:
@CoryMartin-NOAA I will address this comments later. Do you have any simple way to do this? Thanks.

Reply:
Yes with `date`...:
```
GYMD=$(date +%Y%m%d -d "$YY$MM$DD $HH - 6 hours")
GHR=$(date +%H -d "$YY$MM$DD $HH - 6 hours")

RSTDIR=$GDASAPP_TESTDATA/lowres/gdas.$GYMD/$GHR/atmos/RESTART/
```